src/Operations.jl

```diff
@@ -1451,6 +1451,8 @@ function sandbox(fn::Function, ctx::Context, target::PackageSpec, target_path::S
             end
             write_env(temp_ctx.env, update_undo = false)
 
+            allow_autoprecomp && Pkg._auto_precompile(temp_ctx)
```

Comment:
Anyone know why this isn't printing? Is `temp_ctx.io` somehow suppressed?
When precomp is happening here the terminal just pauses after
```
(jl_enhsIi) pkg> test Foo
     Testing Foo
```

Reply:
Yeah
```
temp_ctx.io = Base.DevNull()
```

Reply:
Found it. `Pkg.resolve` has a `Context!` call so the io was being overwritten